Client/Frontend/Browser/Onboarding/OnboardingViewController.swift

```diff
@@ -87,4 +108,17 @@ class OnboardingViewController: UIViewController {
             return label
         }
     }
+    
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        if #available(iOS 13.0, *) {
+            if UITraitCollection.current.userInterfaceStyle != previousTraitCollection?.userInterfaceStyle {
+                theme = OnboardingViewController.getUpdatedTheme()
+                applyTheme(theme)
+            }
+        }
+    }
```

Comment:
@iccub / @jhreis This OK, I'm not super familiar with the theming work

Reply:
This is how it's done everywhere else if you search for `applyTheme`